dc/s2n-quic-dc/src/stream/tests/rpc.rs

```diff
@@ -66,51 +66,67 @@ fn no_loss() {
     sim(|| {
         hello_goodbye();
 
-        {
-            ::bach::net::monitor::on_socket_write(move |write| {
-                let count = COUNT.fetch_add(1, Ordering::Relaxed) + 1;
-                assert!(count <= 4, "flow should only consume 4 packets\n{write:#?}");
-                tracing::info!(?write, "on_socket_write");
-                Ok(())
-            });
-        }
+        ::bach::net::monitor::on_packet_sent(move |packet| {
+            let count = COUNT.fetch_add(1, Ordering::Relaxed) + 1;
+            assert!(
+                count <= 4,
+                "flow should only consume 4 packets\n{packet:#?}"
+            );
+            tracing::info!(?packet, "on_packet_sent");
+            Default::default()
+        });
     });
 
     assert_eq!(COUNT.load(Ordering::Relaxed), 4);
 }
 
 // TODO use this with bach >= 0.0.13
-// Also - this test is broken... need to fix the impl
 #[cfg(todo)]
 #[test]
```

Comment:
Should we drop the cfg Todo?

Reply:
no it's still waiting on https://github.com/camshaft/bach/pull/53. i need to fix a few things with that.